graviti/manager/branch.py

```diff
@@ -79,12 +79,19 @@ def create(self, name: str, revision: Optional[str] = None) -> Branch:
                 the branch name, or the tag name.
                 If the revision is not given, create the branch based on the current commit.
 
+        Raises:
+            ForbiddenError: When create tags on the default branch without commit history.
+
         Returns:
             The :class:`.Branch` instance with the given name.
 
         """
         if not revision:
             revision = self._dataset.HEAD.commit_id
+            if revision is None:
+                raise ForbiddenError(
+                    "Can not create tag on the default branch without commit history"
```

Comment:
I think the word `Can not` is not clear enough.
Will it be better if we use `xxx is not allowed` or `xxx is forbidden` to point out that this behavior is forbidden by design not other reason?